src/styles/ui.css.js

```diff
@@ -359,4 +351,15 @@ export default css`
   margin: 0 0.5em 0.5em;
   border-bottom-width: 1px;
 }
+
+p:is(.respec-hint,.respec-occurrences) {
+  display: block;
+  padding-top: 1em;
```

Comment:
This feels a bit too much to me.
![image](https://user-images.githubusercontent.com/8426945/150900808-796ab8fe-4fa7-48af-a68c-df6c36c371f3.png)

Perhaps:
```suggestion
  margin-top: 0.5em;
```
![image](https://user-images.githubusercontent.com/8426945/150900780-dc11f4f4-7a64-43ed-8b26-dd222aa68150.png)